packages/@react-spectrum/icon/docs/IconTable.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+// eslint-disable-next-line rulesdir/imports
+import {ActionButton} from '@react-spectrum/button';
+// eslint-disable-next-line rulesdir/imports
+import {Cell, Column, Row, Table, TableBody, TableHeader} from '@react-spectrum/table';
+// eslint-disable-next-line rulesdir/imports
+import Copy from '@spectrum-icons/workflow/Copy';
+// eslint-disable-next-line rulesdir/imports
+import js from 'highlight.js/lib/languages/javascript';
+// eslint-disable-next-line rulesdir/imports
+import Lowlight from 'react-lowlight';
+import React, {useEffect, useState} from 'react';
+// eslint-disable-next-line rulesdir/imports
+import {SearchField} from '@react-spectrum/searchfield';
```

Comment:
Wish I didn't have to do this, but `yarn lint` will add these to the icon package dependencies which I believe we don't want seeing as this is docs only. I've tried moving the IconTable into the docs package instead, but ran into a `parcel-packager-ssg: requestAnimationFrame is not defined` error when building the docs that I dunno why is happening.
![image](https://user-images.githubusercontent.com/9661127/85807192-02a0c980-b706-11ea-93f2-8bd58076b8c2.png)